views/challenge/overview.php

```diff
@@ -93,112 +95,122 @@
                         $space = $funding->getSpace()->one();
                         $cover = $funding->getCover();
                         ?>
-                        <a href="<?= $space->createUrl('/xcoin/funding/details', [
-                            'container' => $this->context->contentContainer,
-                            'fundingId' => $funding->id
-                        ]); ?>"
-                            data-target="#globalModal">
+                        <div style="position: relative">
```

Comment:
@rafinala why we have a `style="position: relative"` here , can you ask mortadha how to change this in a css class ?

Reply:
yes morthadha will work on the style for this and market place overview